recipe/meta.yaml

```diff
@@ -20,24 +20,25 @@ build:
     - {{ entrypoint }} = {{ import }}:main
 
 requirements:
-  build:
+  host:
     - python >=3.7
     - pip
 
   run:
     - docutils
+    - packaging
     - pep517
-    - pygments
+    - pygments  # Required internally by docutils for readme check to work
     - python >=3.7
     - python-build
     - requests
-    - setuptools
-    - wheel
+    - setuptools >=61.0.0
+    - trove-classifiers >=2022.6.26
 
 test:
   requires:
+    - flit >=3.4
```

Comment:
Maybe showing my ignorance here, but how is `flit` used for testing?

Reply:
For Pyroma 4.1, it is needed in the test runtime environment as in regebro/pyroma#78 we added a test project that uses it instead of Setuptools as a build backend, so Pyroma could be tested with other backends.

When regebro/pyroma#92 lands presumably in Pyroma 4.2, it will no longer be required (and the deps will be trimmed back and relaxed again in general, including the just-renamed `pep517`), as that modifies Pyroma to use PEP 517 build isolation if the build requirements are not already installed in the runtime environment, and install them automatically.